cherry_picker/test_cherry_picker.py

```diff
@@ -110,7 +111,16 @@ def git_config():
 def tmp_git_repo_dir(tmpdir, cd, git_init, git_commit, git_config):
     repo_dir = tmpdir.mkdir("tmp-git-repo")
     cd(repo_dir)
-    git_init()
+    try:
+        git_init()
+    except subprocess.CalledProcessError:
+        version = subprocess.run(("git", "--version"), capture_output=True)
+        split_version = version.stdout.decode("utf--8").split(".")
+        if int(split_version[0][-1]) < 1 or int(split_version[1]) < 28:
```

Comment:
Maybe this approach is more readable/robust
```suggestion
        # the output looks like "git version 2.34.1"
        v = version.stdout.decode("utf-8").removeprefix('git version ').split('.')
        if (int(v[0]), int(v[1]) < (2, 28):
```